src/dasmon_listener/settings.py

```diff
@@ -45,9 +46,13 @@
 
 MIN_NOTIFICATION_LEVEL = 3
 
-# Import local settings if available
-try:
-    from .local_settings import *  # noqa: F401, F403
-except ImportError:
-    LOCAL_SETTINGS = False
-    pass
+# Try to import local settings from environment
+# brokers
+env_amq_broker = os.environ.get("AMQ_BROKER", None)
+if env_amq_broker:
+    brokers = list(map(tuple, json.loads(env_amq_broker)))  # noqa: F811
+
+# queues
+env_amq_queue = os.environ.get("AMQ_QUEUE", None)
```

Comment:
Similar comment as the one for `brokers`

Reply:
The default value of queue is different from one set in the environment.
I don't know the reason for using these values, and I don't think we should overwrite these default values without knowing the reason behind it